onnxruntime/contrib_ops/cuda/bert/fast_gelu.h

```diff
@@ -18,7 +18,7 @@ class FastGelu final : public CudaKernel {
   Status ComputeInternal(OpKernelContext* ctx) const override;
 
  private:
-  bool use_half2_;
+  bool use_half2_;  // Only applicable to CUDA kernel (not ROCM).
```

Comment:
Instead of comment, use `#ifdef USE_CUDA`

Reply:
OK, let me create a separate fix for that. Thanks.

Reply:
similar reason as above, recommend using `#ifndef USE_ROCM` to avoid undesired hipification.